### ISSUE TITLE
Underbog Heroic pre-nerf

### ISSUE THREAD

**Issue by KimmoKM** [Schaka/TBC-research#7]
"Educated Guess" values have been recorded from Corecraft test server which isn't a primary source (credit for that research goes to NiM and Shiro). However, when I've tried to verify some of their values now and in the past, they've come out pretty much right and in my opinion are preferable to any guessed values and can be used for the time being until primary sources are studied.

It should be noted that Coilfang/Hellfire instance values were recorded with less rigour, such listing damage values as they appeared on Corecraft instead of looking at actual ability behaviour from WDB (and then adjusting it if there's evidence of nerfs) so don't copy them blindly.

|Mob|Damage|Health|Mana|Abilities|Immunities|Source|Comments|
|------|----------|--------|--------|---------|--------------|---------|--------------|
|Underbat|3000-4300|27350|0|Tentacle Lash for 400% damage|?|Educated Guess, [1](http://www.wowhead.com/spell=37956/tentacle-lash#comments:id=218113)|Tentacle Lash only hits targets with their back turned to them|
|Underbog Shambler|4735-6921|21881|6310|Spore Explosion for 2500-3200 and DoT ticking for 450-1000, Allergies, Fungal Regrowth for 7900 heal and 1071 ticks|?|Educated Guess|N/A|
|Underbog Lurker|1950-2850|27350|0|Wild Growth|?|Educated Guess|N/A|
|Bog Giant|4544-6426|62245|0|Growth, Fungal Decay for 200/stack, Trample, Enrage|Immune|WoW Bestiary|N/A|
|Hungarfen|4600-6500|109593|0|Acid Geyser for 2200-2800, Foul Spores for 400-570|Immune|Educated Guess|N/A|
|Underbog Mushroom|N/A|35000|0|Spore Cloud 1250-1500, stacking DoT ticking 500-550|?|Educated Guess|N/A|
|Underbog Frenzy|3550-4750|33986|0|Pierce Armor|?|Educated Guess|N/A|
|Wrathfin Myrmidon|4550-6100|28114|0|Coral Cut ticking 490-750|?|Educated Guess|N/A|
|Wrathfin Sentry|4500-6270|28114|0|Shield Bash, Strike|?|Educated Guess|N/A|
|Murkblood Tribesman|2053-2904|28114|0|Dual Wield, Strike for +165-166, Enrage at 30%|?|Bestiary|N/A|
|Wrathfin Warrior|2245-3050|28114|0|Strike, Shield Bash, Enrage at 30%|?|Educated Guess|N/A|
|Murkblood Oracle|1320-1900|22488|16155|Shadow Bolt for 1770-2300, Corruption ticking 1100, Frost Nova, Amplify Damage, Elemental Armour, Scorch/Frostbolt/Fireball for unknown damage|?|Educated Guess|N/A|
|Murkblood Spearman|1347-1914 melee 2585-3445 ranged|22488|3231|Viper Sting for 2876 mana over 8s, Enrage|?|Educated Guess|N/A|
|Murkblood Healer|1191-1683|22488|16155|Prayer of Healing for 5410-5750, Holy Light for 13900-14700, Renew ticking 1600|?|Bestiary, [1](http://www.wowhead.com/npc=17730/murkblood-healer#comments:id=293152)|N/A|
|Ghaz'an|4182-5914|99630|0|Acid Spit for 1870-2530, stacking Acid Breath for 700 and DoT ticking 170, Tail Sweep for 2025-3375, Enrage at 20%|Immune|Bestiary, [1](http://www.wowhead.com/npc=18105/ghazan#comments:id=24476)|N/A|
|Fen Ray|2080-2663|28114|0|Psychic Horror|?|Educated Guess|N/A|
|Lykul Wasp|?|20259|0|Poison Spit for 850-1120 and DoT ticking for 90|?|Educated Guess|N/A|
|Lykul Stinger|3900-5450|39496|0|Stinger Frenzy for 280-330|?|Educated Guess|N/A|
|Swamplord Musek'ek|2000-2700 ranged, 3200-4000 melee|89224|66180|Dual Wield, Raptor Strike 3900-4600, Knock Away 3200-4200, Multi-Shot, Deterrence, Hunter's Mark, Throw Freezing Trap, Aimed Shot|?|Educated Guess|N/A|
|Claw|1300-1800|135474|0|Maul for +1000, Echoing Roar, Frenzy|?|Educated Guess|N/A|
|Underbog Lord|7705-10200|71810|0|Knock Away for melee damage, Grow, Fungal Decay 150/stack|Immune|Educated Guess|Same as Bog Giants, difficult to determine what Growth stack Wowhead comments were based on. CC values seem plausible|
|The Black Stalker|3909-4111|127505|33090|Levitate, Chain Lightning for 2400-3000, Static Charge ticking 898|Immune|Bestiary|N/A|
|Spore Strider|?|5600|46374|Lightning Bolt for 850-1080|?|Educated Guess|N/A|

**Comment by RealZwox** [comment]
[Bog Giant](http://www.wowhead.com/npc=17723/bog-giant#comments) is prob not Prenerf or wrong. I found some comments on WoWHead

`By Bruciato (4,493 – 1·12·29) on 2007/03/11 (Patch 2.0.10)		
In heroic mode hits for 7.5k on cloths. Base hit damage should be around 9k. Immunity to CC effects are the same of normal dungeon.`

`By huckzilla (118 – 0) on 2008/06/18 (Patch 2.4.2)		
I ran Heroic Underbog last night and one of these bad boys took me out in one hit doing 8.9k physical damage. I'm a level 70 Enhancement Shaman (squid-face), had 8.4k hitpoints buffed, and approximately 4.2k armor. At first, I thought I was lagging because it seemed like a delayed response in the action. Next thing I know, I was pushing up daisies.`

**Comment by KimmoKM** [comment]
I also read those comments and was wondering the same thing. However, I did some sanity checking trying to establish the kind of damage tanks could survive when the giants have a few growth stacks (particularly at the double pull for which not all groups had a suitable kiter) and my interpretation of those comments is that posters were hit with several stacks of growth. Bestiary values (I'm sourcing damage values to Bestiary when they're the same as on Corecraft to make them precise, otherwise there's +-3%ish error) are from 2.3 so huckzilla at 2.4.2 most certainly made this mistake, showing it's clearly possible Bruciato made the same one too.

That's of course assuming Bestiary is correct and I know there has been a few instances when it's not, but human error seems more likely interpretation of evidence to me given that there's no patch notes of Giants being nerfed (2.1.0 mentions Underbog Lord being nerfed, however, but that's already respectable 10k with Knock Away and Growth on top of that).

I probably should have had the rigour to do this in the first place but I now checked some retail videos too and here's [one](https://youtu.be/CRMNLQtsxig?t=1m7s) from 2.0.10 patch and pre-growth damage seems consistent with Bestiary/Corecraft values at first glance (although it's difficult to see exact numbers in grainy video and unknown tank mitigation).

But thanks for the comment, I've found Corecraft has mistakes and I can make mistakes too so a second pair of eyes is always good to have (and perhaps there WAS an undocumented nerf which you could infer from the video I posted or some even earlier ones).

**Comment by RealZwox** [comment]
You are right. I didnt calculated the [Growth](http://www.wowhead.com/spell=40318/growth) in. A Tank would prob not survive 10k raw + 150% fullstacked.

**Comment by RealZwox** [comment]
There is something missing on "Ghaz'an". The Initial Damage of [Acid Breath](http://www.wowhead.com/spell=34268/acid-breath#used-by-npc) :)

**Comment by KimmoKM** [comment]
Added initial damage for Acid Breath.

**Comment by kelno** [comment]
This issue was moved to kelno/TBC-research#8
